lib/calculated.js

```diff
@@ -4,64 +4,108 @@ var debug = require('debug')('loopback-ds-calculated-mixin');
 var _ = require('lodash');
 var Promise = require('bluebird');
 
+function getPropertyConfig(property) {
+  if (typeof property === 'string') {
+    return {
+      callback: property,
+    };
+  }
+  return property;
+}
+
 module.exports = function(Model, options) {
 
   // Trigger a warning and remove the property from the watchlist when one of
   // the property is not found on the model or the defined callback is not found
-  _.mapKeys(options.properties, function(callback, property) {
+  _.mapKeys(options.properties, function(options, property) {
```

Comment:
Should this use `_.forEach` for readability, as we don't return anything but just loop over it?